### ISSUE TITLE
Feature: Add bezier curves to Starling tweens

### ISSUE THREAD

**Issue by tangerinepop** [Gamua/Starling-Framework#891]
Add bezier curves and custom features to Starling tweens similar to what is available through Tweenmax.

**Comment by PrimaryFeather** [comment]
Thanks for the suggestion! I can totally see the sense in that.
Anyone who wants to see this feature, too, add your vote, so that I can prioritize this feature. Thanks in advance!

**Comment by binouze** [comment]
+1 :)

**Comment by 22Round** [comment]
would be grate +1

**Comment by kheftel** [comment]
+1

**Comment by Pandan** [comment]
+1

**Comment by tangerinepop** [comment]
Thanks for the support guys I think this would be a great feature

**Comment by varadig** [comment]
+1

**Comment by PrimaryFeather** [comment]
Thanks for all the feedback, guys! It's settled, then: this feature will be added. :smile:
I can't give any timeframe yet, but I'll update this thread as soon as there is some progress.

**Comment by PrimaryFeather** [comment]
I just noticed that TweenMax is quite flexible when it comes to bezier curves. :wink:
Could you narrow down what exactly you want to do, i.e. describe the most important use-case you want to be able to solve?

**Comment by subdan** [comment]
+1

I want to move object from point A to point B through several points.

**Comment by PrimaryFeather** [comment]
Thanks, @subdan! Please, others that have voted already, describe the actual use-case you want to solve, as well. My guess is that most of you want a method that moves the object through a given list of points?

**Comment by binouze** [comment]
Yes, moving object through a given list of points would be great.
The ability to add callback on each point reached ant to change the easing of the tween on each point would be nice to ;)

**Comment by b005t3r** [comment]
Same here, moving over a list of points is all I need (events dispatched after reaching a given point would be a nice addition as well).
